.circleci/config.yml

```diff
@@ -17,25 +17,19 @@ jobs:
             - v1-dep-{{ .Branch }}-
             - v1-dep-master-
             - v1-dep-
-      - run: 
+      - run:
           name: Bootstrap Carthage
-          command: scripts/bootstrap-if-needed.sh
+          command: scripts/bootstrap.sh
       - save_cache:
           key: v1-dep-{{ .Branch }}-{{ epoch }}
           paths:
             - Carthage
       - run:
-          name: iOS, Swift 4
-          command: set -o pipefail && xcodebuild test SWIFT_VERSION=4.0 -scheme RxStarscream -project RxStarscream.xcodeproj -sdk iphonesimulator -destination "name=iPhone 8" |  xcpretty -c -r html --output $XCODE_TEST_REPORTS/iOS_swift4.html
-      - run:
-          name: iOS, Swift 4.2
-          command: set -o pipefail && xcodebuild test SWIFT_VERSION=4.2 -scheme RxStarscream -project RxStarscream.xcodeproj -sdk iphonesimulator -destination "name=iPhone 8" | xcpretty -c -r html --output $XCODE_TEST_REPORTS/iOS_swift4_2.html
-      - run:
-          name: macOS, Swift 4
-          command: set -o pipefail && xcodebuild test SWIFT_VERSION=4.0 -scheme RxStarscream-macOS -project RxStarscream.xcodeproj -sdk macosx | xcpretty -c -r html --output $XCODE_TEST_REPORTS/macOS_swift4.html
+          name: iOS, Swift 5
+          command: set -o pipefail && xcodebuild test SWIFT_VERSION=5.0 -scheme RxStarscream -project RxStarscream.xcodeproj -sdk iphonesimulator -destination "name=iPhone 8" |  xcpretty -c -r html --output $XCODE_TEST_REPORTS/iOS_swift5.html
       - run:
-          name: macOS, Swift 4.2
-          command: set -o pipefail && xcodebuild test SWIFT_VERSION=4.2 -scheme RxStarscream-macOS -project RxStarscream.xcodeproj -sdk macosx | xcpretty -c -r html --output $XCODE_TEST_REPORTS/macOS_swift4_2.html
+          name: macOS, Swift 5
```

Comment:
Hey @yysskk - just one thing, there's no real reason to stop supporting Swift 4.2.
Xcode 10.2.1 supports both Swift 4.2 and 5, so does RxSwift 5 - I'd like this repo to keep the support & tests for both if possible. Could you do it?

Reply:
@yysskk @freak4pc Any chance we can merge this now? I believe RxSwift 5 specifies Swift 5 as a requirement now, and Xcode 11.2 does not ship with Swift 4.2 support?

Reply:
Dear team,

Did you have any chance to make a decision whether we can merge this PR or not?